spec/spec.md

```diff
@@ -142,6 +172,104 @@ Examples in this document use the Verifiable Credentials Data Model [[spec:VC-DA
   }
 }
 ```
+::: Entity Oriented Example
+
+This pulls the topic to the top of the document, and indexes properties by entity.
+
+```json
+{
+  "id": "32f54163-7166-48f1-93d8-ff217bdb0653",
+  "author": "did:example:alice",
+  "created": "2010-01-01T19:23:24Z",
+  "version": 2,
+  "entity": "did:example:bob",
+  "entries": {
+    "https://example.com/trusted-supplier.schema.json": {
+      "on_time_percentage": 92,
+      "goods": ["applewood", "hotel buffet style", "thick cut"]
+    },
+    "https://example.com/other.schema.json": {
+      "foo": "bar"
+    }
+  }
+}
+```
+::: Entity Topic Oriented Example
+
+This document specifies neither entity or topic at the document level, and indexes by entity, then topic.
+
+```json
+{
+  "id": "32f54163-7166-48f1-93d8-ff217bdb0653",
+  "author": "did:example:alice",
+  "created": "2010-01-01T19:23:24Z",
+  "version": 2,
+  "entries": {
+    "did:example:bob": {
+      "https://example.com/trusted-supplier.schema.json":{
+        "on_time_percentage": 92,
+        "goods": ["applewood", "hotel buffet style", "thick cut"]
+      },
+      "https://example.com/other.schema.json": {
+        "foo": "bar"
+      }
+    },
+    "did:example:carol": {
+      "https://example.com/trusted-supplier.schema.json":{
+        "on_time_percentage": 74,
+        "goods": ["oinkys", "porkys", "wilburs"]
+      },
+      "https://example.com/other.schema.json": {
+        "foo": "baz"
+      }
+    }
+  }
+}
+```
+::: Set Oriented Example
+
+This document specifies neither entity or topic at the document level, and indexes by entity, then topic.
+
+```json
+{
+  "id": "32f54163-7166-48f1-93d8-ff217bdb0653",
+  "author": "did:example:alice",
+  "created": "2010-01-01T19:23:24Z",
+  "version": 2,
+  "entries": [
+    {
+      "schema": "https://example.com/trusted-supplier.schema.json",
+      "entity": "did:example:bob",
+      "properties": {
+        "on_time_percentage": 92,
+        "goods": ["applewood", "hotel buffet style", "thick cut"]
+      }
+    },
+    {
+      "schema": "https://example.com/other.schema.json",
+      "entity": "did:example:bob",
+      "properties": {
+        "foo": "bar"
+      }
+    },
+    {
+      "schema": "https://example.com/trusted-supplier.schema.json",
+      "entity": "did:example:carol",
+      "properties": {
+        "on_time_percentage": 74,
+        "goods": ["oinkys", "porkys", "wilburs"]
+      }
+    },
+    {
+      "schema": "https://example.com/other.schema.json",
+      "entity": "did:example:carol",
+      "properties": {
+        "foo": "baz"
+      }
+    }
+  ]
+}
+```
```

Comment:
Should we have an example where "This document specifies neither entity or topic at the document level, and indexes by topic, then entity."?